docs/en/advanced-concepts/ios-webkit-debug-proxy.md

```diff
@@ -50,6 +50,20 @@ Once installed you can start the proxy with the following command:
 > ios_webkit_debug_proxy -c 0e4b2f612b65e98c1d07d22ee08678130d345429:27753 -d
 ```
 
+You can also set the desired capability called 'startIWDP' to true (https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md). This will run the above command as a subprocess from within Appium and will set the udid. It monitors the proxy for crashes and relaunches when needed.
```

Comment:
maybe explicitly say that if you set the cap to true then you should not start iwdp on your own and we'll do it?

> This will run the above command as a subprocess from within Appium and will set the udid, so you don't have to run `ios_webkit_debug_proxy` yourself anymore.

or similar